<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Polytopal2DWindow</class>
 <widget class="QWidget" name="Polytopal2DWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1553</width>
    <height>992</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>36</red>
        <green>31</green>
        <blue>49</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>54</red>
        <green>46</green>
        <blue>73</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>45</red>
        <green>38</green>
        <blue>61</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>18</red>
        <green>15</green>
        <blue>24</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>24</red>
        <green>21</green>
        <blue>33</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>36</red>
        <green>31</green>
        <blue>49</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>18</red>
        <green>15</green>
        <blue>24</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="127">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>36</red>
        <green>31</green>
        <blue>49</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>54</red>
        <green>46</green>
        <blue>73</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>45</red>
        <green>38</green>
        <blue>61</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>18</red>
        <green>15</green>
        <blue>24</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>24</red>
        <green>21</green>
        <blue>33</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>36</red>
        <green>31</green>
        <blue>49</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>18</red>
        <green>15</green>
        <blue>24</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="127">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>18</red>
        <green>15</green>
        <blue>24</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>36</red>
        <green>31</green>
        <blue>49</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>54</red>
        <green>46</green>
        <blue>73</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>45</red>
        <green>38</green>
        <blue>61</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>18</red>
        <green>15</green>
        <blue>24</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>24</red>
        <green>21</green>
        <blue>33</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>18</red>
        <green>15</green>
        <blue>24</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>18</red>
        <green>15</green>
        <blue>24</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>36</red>
        <green>31</green>
        <blue>49</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>36</red>
        <green>31</green>
        <blue>49</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>36</red>
        <green>31</green>
        <blue>49</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="127">
        <red>18</red>
        <green>15</green>
        <blue>24</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Polytopal2DWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <spacer name="horizontalSpacer_12">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_projectCircles">
       <property name="text">
        <string>Project</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Display Circles</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_displayUnitSphere">
       <property name="text">
        <string>Sphere</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_displayMesh">
       <property name="text">
        <string>Mesh</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_canonizeMesh">
       <property name="text">
        <string>Canonicalize</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_openOBJ">
       <property name="text">
        <string>Open OBJ...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>Export OBJ...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ExportAll">
       <property name="text">
        <string>Export All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ExportSelectedFace">
       <property name="text">
        <string>Export Selected</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_polytopalPlanarControlPoints">
       <property name="text">
        <string>Planar CP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_animations">
       <property name="text">
        <string>Animations</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>Face</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Edge</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Vertex</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_11">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Polytopal2DWindow</receiver>
   <slot>slotIncreaseInversion()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Polytopal2DWindow</receiver>
   <slot>slotDecreaseInversion()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>563</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_projectCircles</sender>
   <signal>clicked()</signal>
   <receiver>Polytopal2DWindow</receiver>
   <slot>slotProjectCirclesClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>506</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>437</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Polytopal2DWindow</receiver>
   <slot>slotDisplayAreaCircles()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>483</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>630</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_displayUnitSphere</sender>
   <signal>clicked()</signal>
   <receiver>Polytopal2DWindow</receiver>
   <slot>slotDisplayUnitSphereChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>554</x>
     <y>506</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>684</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_displayMesh</sender>
   <signal>clicked()</signal>
   <receiver>Polytopal2DWindow</receiver>
   <slot>slotDisplayMeshClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>616</x>
     <y>506</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_canonizeMesh</sender>
   <signal>clicked()</signal>
   <receiver>Polytopal2DWindow</receiver>
   <slot>slotCanonizeMesh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>702</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>726</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_openOBJ</sender>
   <signal>clicked()</signal>
   <receiver>Polytopal2DWindow</receiver>
   <slot>slotOpenOBJFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>788</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>Polytopal2DWindow</receiver>
   <slot>slotExportOBJ()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>874</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>778</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_ExportAll</sender>
   <signal>clicked()</signal>
   <receiver>Polytopal2DWindow</receiver>
   <slot>slotExportAllFaces()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>960</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_ExportSelectedFace</sender>
   <signal>clicked()</signal>
   <receiver>Polytopal2DWindow</receiver>
   <slot>slotExportSelectedFace()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1062</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>827</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Polytopal2DWindow</receiver>
   <slot>slotTypeSelectionChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1294</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>1551</x>
     <y>434</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotOpenOBJFile()</slot>
  <slot>slotExportAllFaces()</slot>
  <slot>slotExportSelectedFace()</slot>
  <slot>slotDisplayUnitSphereChanged()</slot>
  <slot>slotCanonizeMesh()</slot>
  <slot>slotDisplayAreaCircles()</slot>
  <slot>slotDisplayMeshClicked()</slot>
  <slot>slotIncreaseInversion()</slot>
  <slot>slotDecreaseInversion()</slot>
  <slot>slotProjectCirclesClicked()</slot>
  <slot>slotExportOBJ()</slot>
  <slot>slotTypeSelectionChanged(int)</slot>
 </slots>
</ui>
